Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/MandiantAdvantageThreatIntelligence.py

```diff
@@ -1043,6 +1043,7 @@ def get_new_indicators(
             "limit": limit,
             "gte_mscore": minimum_mscore,
             "exclude_osint": exclude_osint,
+            "last_updated": "asc"
```

Comment:
Great catch!
Let's also change the way we save the last_run object
We can delete line 1146 (`date_key = "last_seen" if indicator_type == "Indicators" else "last_updated"`), and change line 1147 to be: `last_run_dict[indicator_type + "LastFetch"] = indicators_list[-1]["last_updated"]`